sycl/unittests/program/KernelRelease.cpp

```diff
@@ -69,15 +69,15 @@ pi_result redefinedKernelGetInfo(pi_kernel kernel, pi_kernel_info param_name,
 }
 
 TEST(KernelReleaseTest, GetKernelRelease) {
-  unittest::PiMock Mock;
-  platform Plt = Mock.getPlatform();
+  platform Plt{default_selector()};
   if (Plt.is_host()) {
     std::cerr << "The program/kernel methods are mostly no-op on the host "
```

Comment:
```suggestion
    std::cout << "The program/kernel methods are mostly no-op on the host "
```
Is skipping the test on the host an error?